pydarn/plotting/rtp.py

```diff
@@ -417,7 +435,9 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
         else:
             tick_interval = 1
         ax.xaxis.set_minor_locator(dates.MinuteLocator(interval=tick_interval))
-        if coord is Coord.SLANT_RANGE:
+        if coord is Coords.SLANT_RANGE:
+            ax.yaxis.set_minor_locator(ticker.AutoMinorLocator(2))
+        if coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
```

Comment:
my fault, sorry!
```suggestion
        elif coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
```

Reply:
or add it to the if condition on the line above (like Marina's comment earlier to be consistent)